wbooks/src/app/components/BackArrowIcon/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { ReactElement } from 'react';
+import { Image, TouchableOpacity } from 'react-native';
+import backArrow from '@assets/General/ic_back.png';
+import { useNavigation } from '@react-navigation/core';
+
+import styles from './styles';
+
+export default function BackArrowIcon(): ReactElement {
```

Comment:
The ReactElement is not needed, it's not wrong, but its not needed.
```suggestion
export default function BackArrowIcon() {
```